lib/publiccloud/aws.pm

```diff
@@ -68,4 +69,11 @@ sub init {
     }
 }
 
+sub get_user_id {
+    zypper_call "in jq";
```

Comment:
I would prefer, this call here https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/publiccloud/prepare_tools.pm#L78

Reply:
ah I wanted to write the same :) so just +1

Reply:
I created this other ticket https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13252